packages/apps/src/modals/AccountChecking.tsx

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017-2020 @polkadot/app-accounts authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import styled from 'styled-components';
+import React from 'react';
+import { Button, Modal } from '@polkadot/react-components';
+import { useTranslation } from '../translate';
+
+interface Props {
+  onClose: () => void;
+  onCreateAccount: () => void;
+  onImportAccount: () => void;
+}
+
+const ModalHeader = styled.div`
+  font-size: 1.2rem;
+  margin-bottom: 1rem;
+  font-weight: bold;
+`;
+
+const Paragraph = styled.div`
+  line-height: 1.5rem;
+`;
+
+export default function AccountCheckingModal ({ onClose, onCreateAccount, onImportAccount }: Props): React.ReactElement {
+  const { t } = useTranslation();
+
+  return (
+    <Modal header={t('Welcome to CENNZnet')}>
+      <Modal.Content>
+        <ModalHeader>{t('You don\'t have any accounts')}</ModalHeader>
```

Comment:
can we make the entries in packages/apps/public/locales/en
so that if we were to support different language, we will have an entry for all the text..

Reply:
Currently I have not added other languages yet, will it throw errors if we don't add translation texts?